api/src/quest/application/combined-course-route.js

```diff
@@ -112,6 +112,12 @@ const register = async function (server) {
           params: Joi.object({
             combinedCourseId: identifiersType.combinedCourseId,
           }),
+          query: Joi.object({
+            page: {
+              number: Joi.number().integer().empty(''),
```

Comment:
**question** : Je trouve ça curieux de mettre un chaîne vide pour un number 🤔 Qu'est ce qui le motive ?

Reply:
rien de spécial, j'avais repris ce qu'on avait fait sur d'autre route
Apres lecture de la doc
> This tells Joi that the empty string should be considered as an empty value

on pourrait modifier 
```suggestion
              number: Joi.number().integer().empty('').default(10),
```